core/src/main/java/org/apache/cxf/message/Message.java

```diff
@@ -196,6 +196,12 @@
      */
     String THREAD_SAFE_STAX_FACTORIES = Message.class.getName() + ".THREAD_SAFE_STAX_FACTORIES";
 
+
+    /**
+     * Integer property to specify how much bytes client must be able to handle.
+     */
+    String MAX_RESPONSE_CONTENT_LENGTH = Message.class.getName() + ".MAX_RESPONSE_CONTENT_LENGTH";
```

Comment:
I think it's probably better to restrict this configuration variable to Netty instead of the CXF Message object, as it only applies to Netty.

Reply:
removed from Message